README.md

```diff
@@ -1,17 +1,80 @@
+
+[![Kubeflow diagram](https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_350,h_304/https://assets.ubuntu.com/v1/10400c98-Charmed-kubeflow-Topology-header.svg)](https://charmed-kubeflow.io)
+
+Charmed Kubeflow is free to use: the solution can be deployed in any environment without constraints, paywall or restricted features. Data labs and MLOps teams only need to train their data scientists and engineers once to work consistently and efficiently on any cloud – or on-premise.
+
+Charmed Kubeflow offers a centralised, browser-based MLOps platform that runs on any conformant Kubernetes – offering enhanced productivity, improved governance and reducing the risks associated with shadow IT.
+
+Learn more about deploying and using Charmed Kubeflow at [https://charmed-kubeflow.io](https://charmed-kubeflow.io).
+
+### Key features
+* Centralised, browser-based data science workspaces: **familiar experience**
+* Multi user: **one environment for your whole data science team**
+* NVIDIA GPU support: **accelerate deep learning model training**
+* Apache Spark integration: **empower big data driven model training**
+* Ideation to production: **automate model training & deployment**
+* AutoML: **hyperparameter tuning, architecture search**
+* Composable: **edge deployment configurations available**
+
+### What’s included in Charmed Kubeflow
+* LDAP Authentication
+* Jupyter Notebooks
+* Work with Python and R
+* Support for TensorFlow, Pytorch, MXNet, XGBoost
+* TFServing, Seldon-Core
+* Katib (autoML)
+* Apache Spark
+* Argo Workflows
+* Kubeflow Pipelines
+
+### Why engineers and data scientists choose Charmed Kubeflow
+* Maintenance: Charmed Kubeflow offers up to two years of maintenance on select releases
+* Optional 24/7 support available, [contact us here](https://charmed-kubeflow.io/contact-us) for more information
+* Optional dedicated fully managed service available, [contact us here](https://charmed-kubeflow.io/contact-us) for more information or [learn more about Canonical’s Managed Apps service](https://ubuntu.com/managed/apps).
+* Portability: Charmed Kubeflow can be deployed on any conformant Kubernetes, on any cloud or on-premise
+
+### Documentation
+Please see the [official docs site](https://charmed-kubeflow.io/docs) for complete documentation of the Charmed Kubeflow distribution.
+
+### Bugs and feature requests
+If you find a bug in our operator or want to request a specific feature, please file a bug here:
+[https://github.com/canonical/dex-auth-operator/issues](https://github.com/canonical/dex-auth-operator/issues)
+
+### License
+Charmed Kubeflow is free software, distributed under the [Apache Software License, version 2.0](https://github.com/canonical/dex-auth-operator/blob/master/LICENSE).
```

Comment:
How about pointing to bundle-kubeflow's [license](https://github.com/canonical/bundle-kubeflow/blob/main/LICENSE)?